app/src/components/common/FeatureNotAvailable.vue

```diff
@@ -2,11 +2,11 @@
   <TmDataMsg icon="timeline">
     <div slot="title">Feature not available</div>
     <div slot="subtitle">
-      This feature hasn't been implemented on this network yet.
+      We haven't implemented this feature for this network yet.
       <br />
       <br />
-      <a class="intercom-button" href="#"
-        >Reach out if you'd like us to enable this feature.</a
+      <a class="intercom-button" @click="$store.dispatch(`displayMessenger`)"
```

Comment:
i actually don't know if this works

Reply:
mmm, I would rather stay safe. You can copy the code from somewhere else. Like this one:

```
    handleIntercom() {
      this.$store.dispatch(`displayMessenger`)
    },
```

You can find it in five different components in our app, so it is sure that this one works

Reply:
`displayMessenger`  only works on mobile, right?

Reply:
> You can find it in five different components in our app

smells funny. we shouldn't have to duplicate a method 5 times. i guess it can be moved to the store?